macOSUpgrade.sh

```diff
@@ -254,14 +263,16 @@ verifyChecksum() {
             downloadInstaller
         fi
     else
-        ##Checksum not specified as script argument, assume true
+        ## Checksum not specified as script argument, assume true
         validChecksum=1
         return
     fi
 }
 
 cleanExit() {
-    kill_process "$caffeinatePID"
+    if [ -n "$caffeinatePID" ]; then
+      kill_process "$caffeinatePID"
+    fi
     ## Remove Script
     /bin/rm -f "$finishOSInstallScriptFilePath" 2>/dev/null
     /bin/rm -f "$osinstallersetupdDaemonSettingsFilePath" 2>/dev/null
```

Comment:
How about those couple `2>/dev/null` ?
They look useless. Why don't you fix those in this PR?

Reply:
Sure. I will change them.

Reply:
I reflected your point https://github.com/kc9wwh/macOSUpgrade/pull/121/commits/4d0fe49d352dde1ef2693d8deee1d7588bde7a6c

Reply:
thanks.